content/04.study.md

```diff
@@ -79,60 +79,64 @@ Moreover, these nonlinear algorithms can deconvolve the effects of multiple vari
+However, *in vivo* TF binding is affected by a variety of other factors beyond sequence affinity, such as competition and cooperation with other TFs, TF concentration, and chromatin state (chemical modifications to DNA and other packaging proteins that DNA is wrapped around) [@doi:10.1016/j.tibs.2014.07.002].
+TFs can thus exhibit highly variable binding landscapes across the same genomic DNA sequence across diverse cell types and states.
+Several experimental approaches such as chromatin immunoprecipitation followed by sequencing (ChIP-seq) have been developed to profile *in vivo* binding maps of TFs [@doi:10.1016/j.tibs.2014.07.002].
+Large reference compendia of ChIP-seq data are now freely available for a large collection of TFs in a small number of reference cell states in humans and a few other model organisms [@tag:Consortium2012_encode].
+Due to fundamental material and cost constraints, it is infeasible to perform these experiments for all TFs in every possible cellular state and species.
+Hence, predictive computational models of TF binding are essential to understand gene regulation in diverse cellular contexts.
+
+Several machine learning approaches have been developed to learn generative and discriminative models of TF binding from *in vitro* and *in vivo* TF binding datasets that associate collections of synthetic DNA sequences or genomic DNA sequences to binary labels (bound/unbound) or continuous measures of binding.
+The most common class of TF binding models in the literature are those that only model the DNA sequence affinity of TFs from *in vitro* and *in vivo* binding data.
+The earliest models were based on deriving simple, compact, interpretable sequence motif representations such as position weight matrices (PWMs) and other biophysically inspired models [@tag:Stormo2000_dna; @doi:10.1093/nar/gkp335; @doi:10.1038/nbt.2486].
+These models were outperformed by general k-mer based models including support vector machines (SVMs) with string kernels [@doi:10.1371/journal.pcbi.1000916; @tag:Ghandi2014_enhanced].
+
+In 2015, Alipanahi et al. developed DeepBind, the first CNN to classify bound DNA sequences based on *in vitro* and *in vivo* assays against random DNA sequences matched for dinucleotide sequence composition [@tag:Alipanahi2015_predicting].
+The convolutional layers learn pattern detectors reminiscent of PWMs from a one-hot encoding of the raw input DNA sequences.
+DeepBind was shown to outperform several state-of-the-art methods from the DREAM5 in vitro TF-DNA motif recognition challenge [@doi:10.1038/nbt.2486].
+Since then, several optimized convolutional and recurrent neural network architectures as well as novel hybrid approaches that combine kernel methods with neural networks have been proposed that further improve performance [@tag:Zeng2016_convolutional; @tag:Lanchantin2016_motif; @arxiv:1706.00125; @doi:10.1101/217257].
```

Comment:
This may be a place to reinsert the RNA binding protein reference:

> Although DeepBind was also applied to RNA-binding proteins, in general RNA binding is a separate problem for which RNA-specific models are being developed [@doi:10.1186/s12859-017-1561-8].
Since then...